README.md

```diff
@@ -654,7 +654,7 @@ See the [Changelog](./CHANGELOG.md) for a list of changes.
 
 ## License
 
-    Copyright (c) 2017, Kinvey, Inc. All rights reserved.
+    Copyright (c) 2018, Kinvey, Inc. All rights reserved.
```

Comment:
If this was a linting error, I guess it will be again, since we are now 2019.

Reply:
Nope, it is not a linting rule.